src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -21,11 +21,11 @@ family:native stack.function:FDebug::CheckVerifyFailedImpl*                v+app
 family:native stack.function:UE::Assert::Private::ExecCheckImplInternal*   v+app
```

Comment:
This rule does not happen on Mac (which uses the cocoa SDK), thus, there's no need to undo the effects of this rule `family:native package:/Users/**  +app`